client/dogs/FormPage.js

```diff
@@ -25,7 +26,8 @@ type State = {
   neighborhood: string,
   zip: string,
   dogName: string,
-  dogAge: string,
+  dogYear: string,
+  dogMo: string,
```

Comment:
Need a more specific name than `Mo`. Abbreviations can be confusing. Best if it describes the type of the string if possible (i.e. is this the name of the month, like "September"?)